girder/taskflow/server/taskflows.py

```diff
@@ -208,7 +213,8 @@ def terminate(self, taskflow, params):
         taskflow_instance.terminate()
 
     @access.user
-    @loadmodel(model='taskflow', plugin='taskflow', level=AccessType.ADMIN)
+    @loadmodel(model='taskflow', plugin='taskflow', level=AccessType.ADMIN,
+               requiredFlags='execute')
```

Comment:
Endpoints are setup in HPCCloud to add flags when sharing but there's no interface for it. At one point I had added automatic group creation per project with the execute flag so you would just add users to that group. 

The idea didn't fly well and so I thought I had removed usage of the accessFlags, but that's obviously not the case. I'm not sure if they restrict anything though? I can run simulations in projects I've created and I'm not applying the execute flag anywhere.